tests/testthat/test-vdiffr_geoms_coords.R

```diff
@@ -1,6 +1,4 @@
-.runThisTest <- Sys.getenv("RunAllseeTests") == "yes"
-
-if (.runThisTest && getRversion() >= "4.1" && require("vdiffr") && require("ggplot2")) {
+if (getRversion() >= "4.1" && require("vdiffr") && require("ggplot2")) {
```

Comment:
make sure all tests are skipped on CRAN, e.g. by adding `.runThisTest` to the if-statement.

Reply:
But they are also skipped locally on my computer, which is suboptimal. I have added `skip_on_cran()` to all these tests, and they are not run on CRAN by default anyway. So we can't be more careful here.